public/app/features/explore/Logs/Logs.tsx

```diff
@@ -1253,7 +1253,7 @@ const getStyles = (theme: GrafanaTheme2, wrapLogMessage: boolean, tableHeight: n
     scrollableLogRows: css({
       overflowY: 'scroll',
       width: '100%',
-      maxHeight: '75vh',
```

Comment:
We've been using 75vh since we originally introduced infinite scrolling, but I just tested with a long screen and we can stretch a big more.